test/e2e_k3d_only/cni/e2e_suite_test.go

```diff
@@ -0,0 +1,17 @@
+package cni_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	"github.com/kumahq/kuma/pkg/test"
+	"github.com/kumahq/kuma/test/e2e_k3d_only/cni"
+)
+
+func TestE2E(t *testing.T) {
+	test.RunSpecs(t, "E2E CNI Suite")
+}
+
+var _ = Describe("Taint controller", Label("arm-not-supported"), cni.AppDeploymentWithCniAndTaintController)
+var _ = Describe("Old CNI", Label("arm-not-supported"), cni.AppDeploymentWithCniAndNoTaintController)
```

Comment:
Instead of having a new family of tests wouldn't a label `kind-not-supported` be a better alternative?

Reply:
Done